qemu/tests/block_hotplug.py

```diff
@@ -37,15 +38,17 @@ def find_disk(vm, cmd):
         """
         Find all disks in guest.
         """
+        session = vm.wait_for_login(timeout=timeout)
         if params.get("os_type") == "linux":
             pattern = params.get("get_disk_pattern", "^/dev/vd[a-z]*$")
         elif params.get("os_type") == "windows":
+            disk_rescan_cmd = params.get("disk_rescan_cmd")
```

Comment:
could you explain why rescan here?

Reply:
Hi coli, please refer to the aliang's last comment, she suggest to replace the time.sleep(5) with rescan operation, due to not sue if 5 seconds is enough for guest rescan, so add a force rescan here.